LabExT/Movement/PathPlanning.py

```diff
@@ -358,7 +378,8 @@ def __init__(self, chip) -> None:
     def set_stage_target(
             self,
             calibration: Type["Calibration"],
-            target: Type[ChipCoordinate]) -> None:
+            target: Type[ChipCoordinate],
+            z_lift: float = None) -> None:
```

Comment:
Do we still need this param? `PathPlanning` should not know anything about lifting/lowering.